instrumentation/kafka/kafka-clients/kafka-clients-2.6/library/src/test/java/io/opentelemetry/instrumentation/kafka/internal/OpenTelemetryMetricsReporterTest.java

```diff
@@ -350,27 +340,9 @@ void observeMetrics() {
                 "kafka.producer.successful_reauthentication_rate",
                 "kafka.producer.successful_reauthentication_total",
                 "kafka.producer.waiting_threads",
-                "kafka.producer.incoming_byte_rate",
-                "kafka.producer.incoming_byte_total",
-                "kafka.producer.outgoing_byte_rate",
-                "kafka.producer.outgoing_byte_total",
-                "kafka.producer.request_latency_avg",
-                "kafka.producer.request_latency_max",
-                "kafka.producer.request_rate",
-                "kafka.producer.request_size_avg",
-                "kafka.producer.request_size_max",
-                "kafka.producer.request_total",
-                "kafka.producer.response_rate",
-                "kafka.producer.response_total",
                 "kafka.producer.byte_rate",
                 "kafka.producer.byte_total",
-                "kafka.producer.compression_rate",
-                "kafka.producer.record_error_rate",
-                "kafka.producer.record_error_total",
-                "kafka.producer.record_retry_rate",
-                "kafka.producer.record_retry_total",
-                "kafka.producer.record_send_rate",
-                "kafka.producer.record_send_total"));
+                "kafka.producer.compression_rate"));
```

Comment:
cc @jack-berg

Reply:
the removed ones were duplicates as these are added to a set removing them shouldn't affect anything

Reply:
I thought I duduped the list but I must have missed something 😅